From e3ef9df6c16f4c76a6496667c85298c36da842dd Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 15 Dec 2023 11:31:46 +0100
Subject: [PATCH 0199/1695] dt-bindings: add TZPROT macro

Add TZPROT macro to define the security level for GPIOs

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 396099fd8c0889fe155e0eb9e5cf6448171b148e dt-bindings: add TZPROT macro]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I23b2f240f38652705a4b77c9e1827a668b05391b
---
 core/include/dt-bindings/gpio/stm32mp_gpio.h | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 core/include/dt-bindings/gpio/stm32mp_gpio.h

diff --git a/core/include/dt-bindings/gpio/stm32mp_gpio.h b/core/include/dt-bindings/gpio/stm32mp_gpio.h
new file mode 100644
index 000000000..ea0466d28
--- /dev/null
+++ b/core/include/dt-bindings/gpio/stm32mp_gpio.h
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: (GPL-2.0-or-later OR BSD-3-Clause) */
+/*
+ * Copyright (C) STMicroelectronics 2023 - All Rights Reserved
+ */
+
+#ifndef _DT_BINDINGS_GPIO_STM32MP_GPIO_H
+#define _DT_BINDINGS_GPIO_STM32MP_GPIO_H
+
+#include <dt-bindings/gpio/gpio.h>
+
+/* Macro to define the security for GPIO */
+#define TZPROT(id) (1 << (id))
+
+#endif
-- 
2.47.2

